src/main/webapp/map.html

```diff
@@ -24,41 +24,47 @@
       }
 
       //initializes the map with specified center and zoom
-      function createRestaurantsMap(){
-        fetch('/Restaurants').then(function(response) {
+      function createCountriesMap(){
+        fetch('/Countries').then(function(response) {
           return response.json();
-        }).then((FamousRestaurants) => {
+        }).then((CountryMaps) => {
+          
           var prev_infowindow = false;
```

Comment:
This should probably be initialized with null instead of false since being null or a valid object seems less surprising than this being a boolean or an object.